.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: true
-language: generic
+language: python
+python:
+  - "3.5"
```

Comment:
We should probably keep Python 2.7 here until we've upgraded at least mutation indexer, possible also datatools.

Reply:
our jenkins has it and I wanted to drop it here. but yeah, makes sense.